### ISSUE TITLE
Add more GTFS editors

### ISSUE THREAD

**Issue by barbeau** [MobilityData/awesome-transit#106]
I just ran across several in this post:
https://gis.stackexchange.com/a/124120/12198

I'm not sure we have all of them on the list.  We need to review and add if they are missing, and somehow distinguish open-source from paid proprietary services.

**Comment by j05u3** [comment]
I agree, also don't forget about [datatools](https://github.com/catalogueglobal/datatools-ui).

**Comment by carlfredl** [comment]
Help wanted: anyone could review this list to see which are still current and draft a PR to add those to the list.

**Comment by etienne0101** [comment]
@carlfredl Just reviewed this list: no resource to add from it (already in the list, or 404)
